website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -0,0 +1,171 @@
+    consul.hashicorp.com/service-meta-bar: baz
+  ```
+
+- `consul.hashicorp.com/service-ignore` - This label can be set on a Kubernetes Service.
+  If set to "true", the service will not be used to register a Consul endpoint. This can be
+  useful in cases where 2 or more services point to the same deployment. Consul cannot register
+  multiple endpoints to the same deployment. This label can be used to tell the endpoint
+  registration to ignore all services except for the one which should be used for routing requests
+  using Consul.
+
+- `consul.hashicorp.com/sidecar-proxy-` - Override default resource settings for
+  the sidecar proxy container.
+  The defaults are set in Helm config via the [`connectInject.sidecarProxy.resources`](/docs/k8s/helm#v-connectinject-sidecarproxy-resources) key.
+
+  - `consul.hashicorp.com/sidecar-proxy-cpu-limit` - Override the default CPU limit.
+  - `consul.hashicorp.com/sidecar-proxy-cpu-request` - Override the default CPU request.
+  - `consul.hashicorp.com/sidecar-proxy-memory-limit` - Override the default memory limit.
+  - `consul.hashicorp.com/sidecar-proxy-memory-request` - Override the default memory request.
+
+- `consul.hashicorp.com/consul-sidecar-` - Override default resource settings for
+  the `consul-sidecar` container.
+  The defaults are set in Helm config via the [`global.consulSidecarContainer.resources`](/docs/k8s/helm#v-global-consulsidecarcontainer) key.
+
+  - `consul.hashicorp.com/consul-sidecar-cpu-limit` - Override the default CPU limit.
+  - `consul.hashicorp.com/consul-sidecar-cpu-request` - Override the default CPU request.
+  - `consul.hashicorp.com/consul-sidecar-memory-limit` - Override the default memory limit.
+  - `consul.hashicorp.com/consul-sidecar-memory-request` - Override the default memory request.
+
+- `consul.hashicorp.com/enable-metrics` - Override the default Helm value [`connectInject.metrics.defaultEnabled`](/docs/k8s/helm#v-connectinject-metrics-defaultenabled).
+- `consul.hashicorp.com/enable-metrics-merging` - Override the default Helm value [`connectInject.metrics.defaultEnableMerging`](/docs/k8s/helm#v-connectinject-metrics-defaultenablemerging).
+- `consul.hashicorp.com/merged-metrics-port` - Override the default Helm value [`connectInject.metrics.defaultMergedMetricsPort`](/docs/k8s/helm#v-connectinject-metrics-defaultmergedmetricsport).
+- `consul.hashicorp.com/prometheus-scrape-port` - Override the default Helm value [`connectInject.metrics.defaultPrometheusScrapePort`](/docs/k8s/helm#v-connectinject-metrics-defaultprometheusscrapeport).
+- `consul.hashicorp.com/prometheus-scrape-path` - Override the default Helm value [`connectInject.metrics.defaultPrometheusScrapePath`](/docs/k8s/helm#v-connectinject-metrics-defaultprometheusscrapepath).
+- `consul.hashicorp.com/service-metrics-port` - Set the port where the Connect service exposes metrics.
+- `consul.hashicorp.com/service-metrics-path` - Set the path where the Connect service exposes metrics.
```

Comment:
## Labels 

- `consul.hashicorp.com/service-ignore` - This label can be set on a Kubernetes Service. 
   If set to "true", the   service will not be used to register a Consul endpoint. This can be
   useful in cases where 2 or more services point to the same deployment. Consul cannot register
   multiple endpoints to the same deployment. This label can be used to tell the endpoint
   registration to ignore all services except for the one which should be used for routing requests
   using Consul.